deploy.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-export "$(grep -v '^#' ".env" | gxargs -d '\n')"
+export $(grep -v '^#' ".env" | gxargs -d '\n')
```

Comment:
is this gxargs a mac-specific command?

Reply:
ah yes, I copied it from helper.sh (which I've changed but not committed) because xargs on mac doesn't have -d option and is not working correctly